platform/darwin/src/MGLStyle.h

```diff
@@ -162,22 +164,46 @@ static const NSInteger MGLStyleDefaultVersion = 9;
  */
 + (NSURL *)satelliteStreetsStyleURLWithVersion:(NSInteger)version;
 
-//- (instancetype)init NS_UNAVAILABLE;
-
-#pragma mark - Runtime style API
+#pragma mark Runtime Styling
 
-- (id <MGLStyleLayer>)layerWithIdentifier:(NSString *)identifier;
+/**
+ Returns a layer that conforms to MGLStyleLayer if any layer with the given identifier was found.
+ 
+ @return layer A layer of type `MGLBackgroundStyleLayer`, `MGLCircleStyleLayer`,
+ `MGLFillStyleLayer`, `MGLLineStyleLayer`, `MGLRasterStyleLayer` or `MGLSymbolStyleLayer`.
```

Comment:
This is how it looks it jazzy now:
![screen shot 2016-08-10 at 3 23 03 pm](https://cloud.githubusercontent.com/assets/1198851/17567904/42565afc-5f0f-11e6-9929-2c2e152c36a6.png)

`@return` does not take a parameter — it should be prose.